### ISSUE TITLE
Create a game bubble on the VITA

### ISSUE THREAD

**Issue by gilgilgan** [PSP-Archive/ARK-4#259]
### New Feature

Standalone ARK-4 is very useful on Vita, but it always has to go through the process of entering the launcher and running the game.

It would be very handy if there was a way to create bubbles with direct access to individual games, like ABMs in Adrenaline.

To clarify to avoid misunderstanding, this is not referring to a custom bubble for ARK-4. 
I'm hoping there's a way to create a bubble for "a PSP game running on top of ARK-4".

**Comment by JoseAaronLopezGarcia** [comment]
Yeah I understand what you mean. It would indeed be very cool, we'll investigate this.
